backend/server.js

```diff
@@ -18,10 +76,80 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const endpoints = expressListEndpoints(app);
+  res.json(endpoints);
 });
 
+//Registration Endpoint
+//http://localhost:9000/register
+app.post("/register", async (req, res) => {
```

Comment:
According to REST naming conventions, endpoints should be named after what they return or in this case create. So this endpoint could be named /users